src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
     public static void main(String[] args) {
+
+
+        Scanner scanner = new Scanner(System.in);
+        TaskList list = new TaskList();
+
+
+
+        horizontalLine();
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello there! I am: \n" + logo + "\nWhat can I help you with!");
+        System.out.println("You can let me know by typing it below, please!");
+        horizontalLine();
+
+        String userInput = scanner.nextLine();
+
+
+
+
+        while (userInput != "bye") {
+            String[] userInputComponents = userInput.split(" ");
+            String requestType = userInputComponents[0];
+            try {
+                Request request = Request.getRequest(requestType);
+
+                switch (request) {
+                    case LIST: {
+                        horizontalLine();
+                        list.printItems();
+                        list.getTaskDetails();
+                        horizontalLine();
+                        userInput = scanner.nextLine();
+                        continue;
+                    }
```

Comment:
the correct indentation can be configured in your IntelliJ ide. it should be something like this:
```suggestion
                switch (request) {
                case LIST:
                    horizontalLine();
                    list.printItems();
                    list.getTaskDetails();
                    horizontalLine();
                    userInput = scanner.nextLine();
                    continue;
```